packages/edit-post/src/components/sidebar/settings-header/style.scss

```diff
@@ -3,8 +3,6 @@
 	padding-left: 0;
 	padding-right: $grid-size-small;
 	border-top: 0;
-	position: sticky;
-	top: 0;
```

Comment:
<img width="304" alt="Capture d’écran 2019-12-05 à 2 44 11 PM" src="https://user-images.githubusercontent.com/272444/70240582-c14acb80-176d-11ea-869f-ca198ca75662.png">

This is not sticky on desktop now. I wonder if it's expected and if it's related to this change here.

Reply:
I have tested and even if unintended this change is good b/c the sticky behaviour of tabs in the sidebar on desktop was a bad peek-a-boo effect.

Reply:
Interesting. Yeah as Andrei suggests, this is what I see:

![current behavior](https://user-images.githubusercontent.com/1204802/70244422-aa5ba780-1774-11ea-9416-de47f06d4735.gif)

This is definitely a separate regression, worth also looking at here. At a glance it looks like a z-index issue.

Before I go and fix it, I wonder why we made those sticky in the first place, I can't recall the circumstances, perhaps because I was up early today :D